src/openzaak/components/autorisaties/admin.py

```diff
@@ -117,6 +117,17 @@ def delete_model(self, request, obj):
         secrets.delete()
         super().delete_model(request, obj)
 
+    @transaction.atomic
+    def delete_queryset(self, request, queryset):
+        """Given a queryset, delete it from the database."""
+        client_ids = queryset.values_list("client_ids", flat=True)
+        client_ids = sum(list(client_ids), [])
```

Comment:
Is this line necessary?

Reply:
Yes, it flats the list
for example, if we delete 2 applicaties, which have client_ids: ['id1', id2'] and ['id3'], then the line123 will give the result:
client_ids = [ ['id1', id2'], ['id3'] ], so we need to flat it further